src/deriving.ml

```diff
@@ -655,5 +674,43 @@ let () =
            ; Context_free.Rule.attr_sig_exception_expect
                Attr.Expect.ec
                expand_sig_exception
+
+           (* equivalent but deprecated attributes *)
```

Comment:
We can just get rid of those. The only two rewriters to interpret `[@@deriving]` attributes are ppxlib and ppx_deriving, and they don't need disambiguation.